spec/shr_core_assessment_vs.txt

```diff
@@ -0,0 +1,28 @@
+Grammar:	ValueSet 5.0
+Namespace:	shr.core
+
+CodeSystem:		LNC = http://loinc.org
+CodeSystem:     SCT = http://snomed.info/sct
+
+ValueSet:       KarnofskyPerformanceStatusVS
+Description:    "Value set for Karnofsky Performance Status, with codes for each decade from 100 to 0. Higher scores are associated with better functional status, with 100 representing no symptoms or evidence of disease, and 0 representing death. This value set corresponds to LOINC list LL4986-7"
+LNC#LA29175-9   "KPS 100: Normal; no complaints; no evidence of disease"
```

Comment:
These do not correspond to the LOINC answer descriptions. Not suggesting we change now, but we need to have a broader conversation about what we include as code descriptions in CIMPL.

Reply:
Yes, but, we really do need to have the numerical score rolled in. Otherwise it would make no sense. Better to make sense than to be enslaved by a tyrannical system that keeps our common in chains.